mathesar_db/tables.py

```diff
@@ -0,0 +1,51 @@
+from sqlalchemy import Column, Integer, String, Table, MetaData, select
+
+from mathesar_db import constants, schemas
+
+DEFAULT_COLUMNS = [
+    Column(constants.ID, Integer, primary_key=True),
+]
+
+
+def create_table(name, schema, column_names, engine):
+    """
+    This method creates a Postgres table corresponding to a collection.
```

Comment:
Please remove "corresponding to a collection" since we've decided not to use "collection" any longer.